### ISSUE TITLE
ReflectionException: Class Symfony\Component\Templating\Helper\Helper not found

### ISSUE THREAD

**Issue by gharlan** [symfony/symfony#23753]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no
| Symfony version  | 3.3.6

I'm using symfony/flex with sf 3.3.
After upgrading to 3.3.6, I get this exception:

![screenshot 2017-08-02 00 15 47](https://user-images.githubusercontent.com/330436/28849611-19b7dafe-7718-11e7-8a6e-7dfb3d45c58e.png)

I do not use the templating component.

I guess it is related to #23573 (ping @nicolas-grekas)

**Comment by tarlepp** [comment]
Yep, I'm having same problem.

**Comment by nicolas-grekas** [comment]
My bad. Should be fixed by #23755. I think that for now, you can opt-out from autowiring and autoconfiguration for the services that need classes the missing classes. Can you confirm (both the fix and that the config change work if possible?)

**Comment by Perf** [comment]
Having this problem.
```
Executing script assets:install --symlink --relative public [KO]
 [KO]
Script assets:install --symlink --relative %PUBLIC_DIR% returned with error code 255
!!
!!  Fatal error: Uncaught ReflectionException: Class Symfony\Component\Templating\Helper\Helper not found in /home/vagrant/Code/vendor/symfony/security-bundle/Templating/Helper/LogoutUrlHelper.php:23
```
composer.json
```
"require": {
        "php": "^7.1.3",
        "api-platform/api-pack": "^2.1@dev",
        "api-platform/core": "^2.1@dev",
        "doctrine/doctrine-migrations-bundle": "^1.2",
        "javiereguiluz/easyadmin-bundle": "^1.16",
        "symfony/console": "^3.3",
        "symfony/flex": "^1.0",
        "symfony/framework-bundle": "^3.3@dev",
        "symfony/yaml": "^3.3"
    },
```
`symfony/framework-bundle (3.3.x-dev 25bc5e5)`

So, issue seems not to be fixed yet. Or I'm doing something wrong.

**Comment by jvasseur** [comment]
@Perf the issue was in the dependency injection component not the framework bundle.

**Comment by chalasr** [comment]
In the config component in fact.

**Comment by Perf** [comment]
Thank you guys!
Btw, it was fixed by `composer req config:^3.3@dev`, got it from #23913
